_episodes/01-introduction.md

```diff
@@ -1,11 +1,42 @@
 ---
 title: "Introduction"
-teaching: 0
+teaching: 10
 exercises: 0
 questions:
-- "Key question"
+- "Why are best practices necessary in research software?"
+- "How Open Source can help with better quality of software?"
 objectives:
-- "First objective."
+- "Basics of Open Science in research software"
+- "Introduction to the FAIR principles"
```

Comment:
Given that this introduction is for all four of the best practices, I think that it may be too specific to have a question just for metadata. Thoughts?

Reply:
Totally agree, you are right, no need to add anything here